### PR TITLE
Added Java invokeModel REST Api example

### PR DESCRIPTION
**Pull request by maxtybar**
*Issue #, if available:* N/A

*Description of changes:* Added Java REST Api code sample for `invokeModel` Amazon Bedrock API.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: introduction-to-bedrock/java/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+## Contents
+
+- [Run invokeModel API with Java](./my-app/src/main/java/com/mycompany/app/App.java) - Call invokeModel Amazon Bedrock REST API and sing it with AWSSigV4 using Java programming language
+
+## Getting Started
+
+To get started with the samples, follow these steps:
+
+1. Clone the repository: `git clone https://github.com/your-username/your-repo.git`
```
Comment (aarora79) [inline introduction-to-bedrock/java/README.md:13]: Please change the clone command to be the exact repo link for this repo.

**Review by aarora79 — COMMENTED** [review]
Please update the git clone command, see comment.

**Review by aarora79 — CHANGES_REQUESTED** [review]
please update the code repo link in readme.

File: introduction-to-bedrock/java/README.md (hunk 1 above)
Comment (maxtybar) [reply introduction-to-bedrock/java/README.md]: Updated.

**Review by aarora79 — APPROVED** [review]

**Pull request merged**